### DISCUSSION TITLE
Abyss islands covered by smoke

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#13930] [Bug reports]
### Disclaimers  

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

So you can't see and explore them
You might say you shouldn't because it's hot there, but even basic engineer jumpsuit can block that heat

https://github.com/FakeFishGames/Barotrauma/assets/122838333/fb76f04d-41a4-4bba-891a-ef50bef946f8

https://github.com/FakeFishGames/Barotrauma/assets/122838333/ae849c41-bfd3-4af4-ac9d-535ca6e80712

Yes, ok, it might be too hot, and you most certainly won't have pucs in second biome, but still, sprite depth is funny
![20240522071016_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/b4a71f9e-c5c0-4898-acc5-79360a583f96)

### Reproduction steps

Set particle count to max
Go to Europan ridges abyss and check the islands

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
Thank you for your report, we're aware of the issue and are working on a fix.

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/pull/5393 (PR in out private repo)
